cmd/grant/user/cmd.go

```diff
@@ -87,9 +87,9 @@ func run(_ *cobra.Command, argv []string) {
 		os.Exit(1)
 	}
 	if username == idp.ClusterAdminUsername {
-		r.Reporter.Errorf("Username '%s' is not allowed. It's must in '%s' group. "+
+		r.Reporter.Errorf("Username '%s' is preserved for `rosa create/delete admin` command. "+
```

Comment:
Should the term be reserved instead of preserved?

Reply:
@gdbranco 
thanks for the commments
can you take another look on the fixed one? Thanks